locales/base/translation.json

```diff
@@ -842,7 +842,7 @@
     "titleGuided": "Welcome to {{appName}}!\nLet’s create your crypto wallet.",
     "createAccount": "Create new account",
     "restoreAccount": "Restore my account",
-    "createNewWallet": "Create new wallet",
+    "createNewWallet": "Create new account",
```

Comment:
I think we should hold off on changing wallet -> account, or if we intend to do this, we should also change the hasWallet copy? Last I checked with product, this wasn't confirmed

Reply:
Yeah, will revert this for the time being.